avocado/core/job.py

```diff
@@ -539,6 +542,9 @@ def run(self):
                  :mod:`avocado.core.exit_codes` for more information.
         """
         runtime.CURRENT_JOB = self
+        for ext in self.job_pre_post_dispatcher.extensions:
```

Comment:
Yep, I agree with the "pre-test" definition and it should be part of the documentation ...